# An array of detections (each detection has its own header)
Detection[] detections
